swagger2.0/oic.r.rule.inputcollection.swagger.json

```diff
@@ -0,0 +1,213 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OCF Rule Input Collection",
+    "version": "20190910",
+    "license": {
+      "name": "OCF Data Model License",
+      "url": "https://github.com/openconnectivityfoundation/core/blob/e28a9e0a92e17042ba3e83661e4c0fbce8bdc4ba/LICENSE.md",
+      "x-copyright": "copyright 2019 Open Connectivity Foundation, Inc. All rights reserved."
+    },
+    "termsOfService": "https://openconnectivityfoundation.github.io/core/DISCLAIMER.md"
+  },
+  "schemes": ["http"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/RuleInputCollectionResURI?if=oic.if.ll": {
+      "get": {
+        "description": "Collection of Links to the Resources (i.e., Rule Inputs) that contain the Properties whose values are evaluated as part of the Rule Expression.",
+        "parameters": [
+          {"$ref": "#/parameters/interface-all"}
+        ],
+        "responses": {
+          "200": {
+            "description" : "Retrieves the rule input Links.",
+            "x-example":
+            [
+              {
+                "anchor": "timeperiod",
+                "href": "/mytimeperiodresource",
+                "rel": ["ruleinput"],
+                "rt":   ["oic.r.time.period"],
+                "if":   ["oic.if.a","oic.if.baseline"],
```

Comment:
Only list the OCF Interface used?  From the CR:

The Link has an "if" set to the "if" used for the internal "observe" of the input Resource.

Reply:
True.  I haven't revisited the PR in a while.  I'll do a scrub of all of this.